### PR TITLE
Modification of the error handling for the name of the house on the front side

### PR DESCRIPTION
**Pull request by Terdious**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [x] Is the linter passing? (`npm run eslint` on both front/server)
- [x] Did you run prettier? (`npm run prettier` on both front/server)

NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open.

### Description of change

To follow up on issue #1055

### CODE REVIEW

File: front/src/actions/house.js [hunk 1]
```diff
@@ -233,6 +233,12 @@ function createActions(store) {
               [house.id]: RequestStatus.RoomValidationError
             }
           });
+        } else if (status === 422 && url.includes('/house')) {
+          store.setState({
+            houseUpdateStatus: {
+              [house.id]: RequestStatus.HouseValidationError
```
Comment (atrovato) [inline front/src/actions/house.js:239]: Is there really a difference with RequestStatus.ValidationError?

Comment (Terdious) [reply front/src/actions/house.js:239]: Yes, it is different @atrovato. Look at the issue I specified in the PR description (#1055). When you fill in the field with a name that is too long, the RequestStatus.ValidationError error message returns that the name of the house cannot be empty.
image

Comment (atrovato) [reply front/src/actions/house.js:239]: I think you only have to change to current message to inform both information: name is mandatory and max size is N chars.
And maybe limit the name input field to N chars.

On this page, if you empty the name, you will get a 422 too.

**Comment by Pierre-Gilles** [comment]
Fixed in https://github.com/GladysAssistant/Gladys/commit/183742e610a37b0fc33065b43f0d5c8b0e82ff0d

**Pull request closed without merging**